tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php

```diff
@@ -392,7 +392,7 @@ public function testSubmitWebform() {
     $this->assertSession()->waitForField('properties[options][options][civicrm_option_1][label]');
     $this->getSession()->getPage()->fillField('properties[options][options][civicrm_option_1][label]', 'Red - Recommended');
     $this->htmlOutput();
-    $this->createScreenshot($this->htmlOutputDirectory . '/afterlabelchange.png');
+    // $this->createScreenshot($this->htmlOutputDirectory . '/afterlabelchange.png');
     $this->getSession()->getPage()->pressButton('Save');
```

Comment:
Why am I still getting an afterlabelchange.png in my artifacts? 

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/5340555/132263447-4cb408f0-09c2-4fcc-a1d4-a5190d339e17.png">